CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.0.0] Stable Release
+### Added
+- Added 'Automatic-Module-Name' manifest entry to resulting jre10 jars such that driver module 'com.microsoft.sqlserver.jdbc' will be accessible to JDK 10 customers [#732](https://github.com/Microsoft/mssql-jdbc/pull/732)
```

Comment:
Technically correct but a little awkward, maybe "Added 'Automatic-Module-Name' manifest entry, allowing jre10 customers to access driver module 'com.microsoft.sqlserver.jdbc'"?

Reply:
I'm fine either way.. will make the change.